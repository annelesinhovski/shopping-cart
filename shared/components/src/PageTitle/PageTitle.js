import React, { Fragment } from 'react';
import Head from 'next/head';

export const PageTitle = ({ title, children }) => (
  <Fragment>
    <Head>
      <title>{title}</title>
    </Head>
    {children}
  </Fragment>
);
