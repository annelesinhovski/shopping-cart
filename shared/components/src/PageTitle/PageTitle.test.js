import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { PageTitle } from './PageTitle';

test('Check if the child is rendered', () => {
  const { asFragment } = render(<PageTitle>Teste</PageTitle>);
  expect(asFragment).toMatchSnapshot();
});
