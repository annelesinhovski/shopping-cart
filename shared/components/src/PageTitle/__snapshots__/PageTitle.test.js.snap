// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if the child is rendered 1`] = `[Function]`;
